src/components/illustration/illustration-set.ts

```diff
@@ -0,0 +1,9 @@
+import * as LIST from './illustration-files';
+export default LIST;
+
+/**
+ * This file makes the given set of svg files available
+ * to the codebase.
+ */
+export type IllustrationType = keyof typeof LIST;
+export const ILLUSTRATION_NAMES = Object.keys(LIST) as IllustrationType[];
```

Comment:
Very useful type magic in here. Approved 👍

Reply:
Thanks!

What I'm still missing is a typed hash. Right now it's easy to use the typed props
but when UI want to access it somewhere else, like in the story in only have this big fat array of keys `ILLUSTRATION_NAMES` which I can access `ILLUSTRATION_NAMES[0]` but I would rather have something like the original export `ILLUSTRATION_HASH.blimp_001` (which should then return a string `blimp_001`) and such. I had a reduce in there but I wasn't able to quickly type it appropriately. So there is a challenge left 😁🤘